private/pkg/app/appcmd/webpages.go

```diff
@@ -0,0 +1,305 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package appdoc provides a markdown generator for cobra commands.
+// In the future this will need to be adapted to appcmd.Command when we remove usage of Cobra.
+package appcmd
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"html"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"unicode"
+
+	"github.com/bufbuild/buf/private/pkg/app"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var codeBlockRegex = regexp.MustCompile(`(^\s\s\s\s)|(^\t)`)
+
+const (
+	slugPrefixFlagName      = "slug-prefix"
+	excludeCommandsFlagName = "exclude-command"
+)
+
+type flags struct {
```

Comment:
this should be named `webpagesFlags` or the like, this isn't `flags` for all of `appcmd`

Reply:
Resolved: 09b324a8b1a437b7224876cd5465c86ff5d3fbb4